integration-tests/src/test/kotlin/com/google/devtools/ksp/test/KSPCmdLineOptionsIT.kt

```diff
@@ -29,7 +29,8 @@ class KSPCmdLineOptionsIT(val useKSP2: Boolean) {
         gradleRunner.withArguments("clean", ":processors:build").build()
         val processorJar = File(project.root, "processors/build/libs/processors-1.0-SNAPSHOT.jar")
         val classLoader = URLClassLoader(arrayOf(processorJar.toURI().toURL()), javaClass.classLoader)
-        val compiler = classLoader.loadClass(K2JVMCompiler::class.java.name).newInstance() as K2JVMCompiler
+        val compiler = classLoader.loadClass(K2JVMCompiler::class.java.name).getDeclaredConstructor()
```

Comment:
`newInstance` is deprecated in favor for `getDeclaredConstructor().newInstance()`